### PR TITLE
Production changes for 0421 release

### PR DESCRIPTION
**Pull request by jeffchi**
- workload image is `1.3.0.10`
- git branch from `master` to `main`
- volume name must be `hpsb`
- new update section in sbs for vpc

### CODE REVIEW

File: SBS-HPVScloud.md [hunk 1]
```diff
@@ -654,8 +653,7 @@ Created               2021-12-06
 ```
 
 8. Update the following parameters of the `sbs-config.json` configuration file:
-   - "build_image_tag": "1.3.0.9"
-   - "RUNTIME_TYPE": "classic"
```
Comment (manojgollaprolu21) [inline SBS-HPVScloud.md:658]: Why is classic removed ? please add it

File: SBS-VPC.md [hunk 2]
```diff
@@ -559,3 +557,46 @@ Complete the following steps:
...
+
+## How to update the Secure Build Server (SBS)
+
+Complete the following steps:
+
+1. Export the state image to the Cloud Object Storage (COS) by folloing the [instructions](SBS-VPC.md#how-to-get-the-state-image).
+
+2. Generate the new contract with both the `workload` and `env` sections. Use the `workload` section on [this page](https://cloud.ibm.com/docs/vpc?topic=vpc-about-hpsb#hpvs_hpsb), and in the `env` section, make sure that you change the volume name from `test` to `hpsb`. For more information about creating the contract, see [About the contract](https://cloud.ibm.com/docs/vpc?topic=vpc-about-contract_se).
+
+3. Create a new SBS server as mentioned in the section [Deploying the Secure Build Server](SBS-HPVScloud.md#deploying-the-secure-build-server) by using the contract file.
```
Comment (shelly-yadav) [inline SBS-VPC.md:570]: This is pointing to SBS-HPVScloud.md. But it should point to SBS-VPC.md

File: SBS-HPVScloud.md [hunk 3]
```diff
@@ -43,12 +43,11 @@ Create the `sbs-config.json` file in any location you choose on your local machi
 ```
 {
   "HOSTNAME": "sbs.example.com",
-  "RUNTIME_TYPE": "classic",
```
Comment (shelly-yadav) [inline SBS-HPVScloud.md:46]: This needs to be there in the document . It should be present

**Review by manojgollaprolu21 — APPROVED** [review]
LGTM

**Pull request merged**